flags/flags.go

```diff
@@ -195,6 +195,21 @@ $STEPPATH/config/ca.json`,
 be stored in the 'x5c' header.`,
 	}
 
+	// X5tCert is a cli.Flag used to pass the x5t header certificate thumbprint
+	// for a JWS or JWT.
+	X5tCert = cli.StringFlag{
+		Name:  "x5t-cert",
+		Usage: "Certificate in PEM format to use for the 'x5t' header of a JWS or JWT",
+	}
+
+	// X5tKey is a cli.Flag used to pass the private key (corresponding to the x5t-cert)
```

Comment:
I'm not seeing this variable used anywhere. I guess it's nice to have it defined? _shrug_